docs/quickstarts/insights-inventory-workspace-rbac/insights-inventory-workspace-rbac.yaml

```diff
@@ -44,28 +44,28 @@ spec:
+              1. **inventory:workspace:write** → Allows users to edit inventory workspace membership (add and remove systems from workspaces).
         1. Select inventory permissions as needed. Here are some possible variations: <br>
-            - If you want to give a user full access to the group and its systems → select all 4 permissions.
-            - If you want to give a user full access to the systems inside a group → but not group editing access, give them inventory:hosts:read, inventory:hosts:write, and inventory:group:read, but NOT inventory:group:write.
-            - If you want to give a user full access to ungrouped systems → select all 4 permissions (ungrouped systems are considered a group).
+            - If you want to give a user full access to the workspace and its systems → select all 4 permissions.
+            - If you want to give a user full access to the systems inside a workspace → but not workspace editing access, give them inventory:hosts:read, inventory:hosts:write, and inventory:workspace:read, but NOT inventory:workspace:write.
+            - If you want to give a user full access to ungrouped systems → select all 4 permissions (ungrouped systems are considered a workspace).
         1. Click **Next**.
-        1. On Step 3 (Define inventory group access), for each permission, select groups you would like to apply the permissions to using the dropdown menus. <br>
+        1. On Step 3 (Define inventory workspace access), for each permission, select workspaces you would like to apply the permissions to using the dropdown menus. <br>
         
             Possible scenarios include:
-            - You would like to give users the ability to only see systems in specific groups and NOT ungrouped systems → Select those groups only.
-            - You would like to give users the ability to see systems in specific groups AND any ungrouped systems → Select those groups for all permissions AND “ungrouped systems" for inventory:hosts permissions.
+            - You would like to give users the ability to only see systems in specific workspaces and NOT ungrouped systems → Select those workspaces only.
+            - You would like to give users the ability to see systems in specific workspaces AND any ungrouped systems → Select those workspaces for all permissions AND “ungrouped systems" for inventory:hosts permissions.
             - You would like to give users the ability to see everything in the inventory → There is no need to create a custom role then. You can cancel the creation flow.
         1. Once you’ve made your selections, click **Next**.
         1. Review your selections. Click **Submit**.
 
-        Repeat this process for every Inventory group or groups of users you want to give specific group access to.
+        Repeat this process for every Inventory workspace or workspaces of users you want to give specific workspace access to.
```

Comment:
should be lower case inventory, and not "workspaces of users", but "groups of users" hahaha. So to recap, this sentences should say:

"Repeat this process for every inventory workspace or group of users you want to give specific workspace access to."